python/lsst/dax/apdb/apdbSchema.py

```diff
@@ -109,7 +112,7 @@ def make_minimal_dia_source_schema():
 
 
 @compiles(CreateTable, "oracle")
-def _add_suffixes_tbl(element, compiler, **kw):
+def _add_suffixes_tbl(element: Any, compiler: Any, **kw: Any) -> str:
```

Comment:
They are indeed, but `mypy` has no way to verify that because SQLAlchemy does not provide `mypy` interfaces (and I was too lazy to check the specific type of these objects).